tests/lib/snaps/test-snapd-netlink-audit/bin/bind

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+ 
+import socket
+import os
+
+
+if __name__ == "__main__":
+    # Create Netlink socket
+    # For more information about protocols see https://github.com/torvalds/linux/blob/master/include/uapi/linux/netlink.h
+    netlink_audit_protocol = 9
+    cn_idx_proc = 1
+
+    with socket.socket(socket.AF_NETLINK, socket.SOCK_DGRAM, netlink_audit_protocol) as sock:
+        sock.bind((os.getpid(), cn_idx_proc))
+        print('netlink connect bound')
```

Comment:
I think the print is stale, perhaps `netlink audit bound`?

Reply:
done